packages/mcp-server-supabase/src/tools/database-operation-tools.ts

```diff
@@ -213,6 +216,47 @@ export function getDatabaseTools({
           throw new Error('Cannot apply migration in read-only mode.');
         }
 
+        // Try to request user confirmation via elicitation
+        if (server) {
+          try {
+            const result = (await server.request(
```

Comment:
Wondering if we can use the [elicitInput](https://github.com/modelcontextprotocol/typescript-sdk?tab=readme-ov-file#eliciting-user-input) helper here